src/commands/connections/util.ts

```diff
@@ -0,0 +1,33 @@
+export const outputFormat = {
+	table: [
+		{
+			header: 'ID',
+			property: 'id'
+		},
+		{
+			header: 'Name',
+			property: 'name'
+		},
+		{
+			header: 'Platform',
+			property: 'platform.name'
+		},
+		{
+			header: 'Enabled',
+			property: 'disabled',
+			transform: val => !val
+		},
+		{
+			header: 'IsPool',
+			property: 'group'
+		},
+		{
+			header: 'ParentPool',
+			property: 'pool.name',
```

Comment:
can a connection be  a part of more than one pool?

Reply:
No, a connection can only be part of 1 pool